jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/PluginConfigurationProcessor.java

```diff
@@ -42,7 +43,9 @@
 
   /**
    * Gets the value of the {@code <container><appRoot>} parameter. Throws {@link
-   * MojoExecutionException} if it is not an absolute path in Unix-style.
+   * MojoExecutionException} if it is not an absolute path in Unix-style. If this parameter is null,
+   * {@code JavaLayerConfigurations.DEFAULT_WEBAPP_ROOT} is returned for war packaging, {@code
```

Comment:
`{@code` --> `{@link`

`.DEFAULT_` --> `#DEFAULT_`
`WEBAPP` --> `WEB_APP`
`for war packaging` --> `for WAR packaging`

Reply:
Done